SConstruct

```diff
@@ -3208,6 +3188,11 @@ if doConfigure :
 		alembicEnv.Prepend(
 			CPPFLAGS = "-DIECOREALEMBIC_WITH_OGAWA"
 		)
+		alembicEnv.Append(
+			LIBS = [
+				"Alembic$ALEMBIC_LIB_SUFFIX",
+			],
```

Comment:
The branch below this for Alembic < 1.6 will be broken now, because it's not adding `AlembicAbcGeom`. I don't think we need that branch though, so let's just remove it.

Reply:
Addressed in 7a656ea5417653e52ad5d97fa4967cea8e8e5b38